dreambooth.py

```diff
@@ -380,9 +380,15 @@ def __init__(
         self.instance_images_path = []
         self.class_images_path = []
 
+        def extract_concept(fn):
+            return fn.stem.replace("_", " ").split("-")[0]
```

Comment:
```suggestion
        def extract_concept(filename):
            return filename.stem.replace("_", " ").split("-")[0]
```

Renaming to avoid confusion about `fn` which is often used a shorthand for `function`